File: app/src/main/java/com/udacity/astroapp/utils/WebIntentUtils.java
package com.udacity.astroapp.utils;

import android.content.Context;
import android.content.Intent;
import android.net.Uri;

public class WebIntentUtils {

    public static void openWebsiteFromStringUrl(Context context, String url) {
        Uri uri = Uri.parse(url);
        Intent openAsteroidDetailsIntent = new Intent(Intent.ACTION_VIEW);
        openAsteroidDetailsIntent.setData(uri);
        context.startActivity(openAsteroidDetailsIntent);
    }
}

File: app/src/main/kotlin/com/udacity/astroapp/utils/WebIntentUtils.kt
package com.udacity.astroapp.utils

import android.content.Context
import android.content.Intent
import android.net.Uri

object WebIntentUtils {

    fun openWebsiteFromStringUrl(context: Context, url: String) {
        val uri = Uri.parse(url)
        val openWebsiteIntent = Intent(Intent.ACTION_VIEW).apply { data = uri }
        context.startActivity(openWebsiteIntent)
    }
}
